### PR TITLE
Remove http(s):// from URL

### PR DESCRIPTION
**Pull request by sdawka**

### CODE REVIEW

**Comment by sdawka** [comment]
> Maybe we can merge both and use a regular expression =)

Is this good, or could it be more elegant?

File: modules/url-check.js [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
+const regex = /http[s]?:\/\//g
 module.exports = () => {
   if (!url) {
     console.log(red('No website was provided.'))
     process.exit(1)
   }
-  if (url.startsWith('http://')) {
-    url = url.slice(7)
+  if (url.match(regex)) {
+    url = url.replace(url.match(regex)[0],'')
```
Comment (DanielRuf) [inline modules/url-check.js]: This can be simplified, see https://codepen.io/DanielRuf/pen/eYYRNYz?editors=1111

File: modules/url-check.js [hunk 2]
```diff
@@ -1,8 +1,11 @@
 const { red } = require('kleur')
-
+const regex = /http[s]?:\/\//g
```
Comment (DanielRuf) [inline modules/url-check.js]: Please add the empty line before it.

File: modules/url-check.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 const { red } = require('kleur')
+
 const regex = /http[s]?:\/\//g
```
Comment (DanielRuf) [inline modules/url-check.js]: `/g` global flag is not needed,

File: modules/url-check.js [hunk 4]
```diff
@@ -1,12 +1,10 @@
 const { red } = require('kleur')
 
-const regex = /http[s]?:\/\//g
+const httpsRegex = /http[s]?:\/\//g
```
Comment (DanielRuf) [inline modules/url-check.js]: Please use /^https?.../ (^for the beginning of the input).

File: modules/url-check.js [hunk 5]
```diff
@@ -1,8 +1,10 @@
 const { red } = require('kleur')
 
+const httpsRegex = /https?:\/\//
```
Comment (DanielRuf) [inline modules/url-check.js]:
```suggestion
const httpsRegex = /^https?:\/\//
```

Comment (DanielRuf) [reply modules/url-check.js]: And please move it into the exported body. See cli-help.js for example. Only module / dependency imports should be outside.

**Comment by DanielRuf** [comment]
Hey @sdawka,

thanks for your contribution to website-checks.
I have sent you an invitation as collaborator with a few more rights.

You can accept it at https://github.com/DanielRuf/website-checks/invitations

**Pull request merged**